src/ui/page_storage.c

```diff
@@ -567,13 +567,13 @@ page_pack_t pp_storage = {
     .on_right_button = page_storage_on_right_button,
     .post_bootup_run_priority = 50,
     .post_bootup_run_function = page_storage_init_auto_sd_repair,
-    .post_bootup_run_complete = page_storage_is_sd_repair_complete,
```

Comment:
This function needs to be called otherwise it may break the input_thread which monitors the SD Card mounting if I'm not mistaken.

Reply:
You're absolutely right and I actually missed that this method has side effects. I'll fix that in a minute!